x-pack/plugins/maps/public/components/layer_addpanel/flyout_footer/view.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyoutFooter,
+  EuiButtonEmpty,
+  EuiButton,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+
+export const FlyoutFooter = (
+  { onClick, disableButton, buttonText, closeFlyout, hasLayerSelected, isLoading }
```

Comment:
`disableButton` is vague because the footer has multiple buttons. Maybe use a clearer name like `disableNextButton`